components/dashboard/src/repositories/detail/prebuilds/PrebuildDetailPage.tsx

```diff
@@ -0,0 +1,148 @@
+            case PrebuildPhase_Phase.ABORTED:
+                return {
+                    icon: <CircleSlash2Icon size={20} className="text-gray-500" />,
+                    description: "Prebuild aborted",
+                };
+            case PrebuildPhase_Phase.TIMEOUT:
+                return {
+                    icon: <CircleSlash2Icon size={20} className="text-gray-500" />,
+                    description: "Prebuild timeout",
+                };
+            case PrebuildPhase_Phase.AVAILABLE:
+                return {
+                    icon: <CheckCircle2Icon size={20} className="text-green-500" />,
+                    description: "Prebuild available",
+                };
+            case PrebuildPhase_Phase.FAILED:
+                return {
+                    icon: <AlertTriangleIcon size={20} className="text-kumquat-base" />,
+                    description: "Prebuild available",
+                };
+
+            case PrebuildPhase_Phase.UNSPECIFIED:
+            default:
+                return {
+                    icon: <CircleSlash2Icon size={20} className="text-gray-500" />,
+                    description: "Prebuild unknown",
+                };
+        }
+    }, [prebuild?.status?.phase?.name]);
+
+    return (
+        <div className="w-full">
+            <BreadcrumbNav
+                pageTitle="Prebuild history"
+                pageDescription={
+                    <>
+                        <span className="font-semibold">{repository?.name || ""}</span>{" "}
+                        <span className="text-pk-content-secondary">{prebuild?.ref || ""}</span>
+                    </>
+                }
+                backLink="/repositories"
+            />
+            {(repositoryIsLoading || prebuildIsLoading) && <Loader2 className="animate-spin" />}
+            {prebuild && (
+                <div className="app-container mb-8">
+                    <div className={"border border-pk-border-base rounded-xl py-6 divide-y"}>
+                        <div className="px-6 pb-4">
+                            <div className="flex flex-col gap-2">
+                                <div className="flex justify-between">
+                                    <div className="font-semibold text-pk-content-primary">
+                                        {prebuild.commit?.message}
+                                    </div>
+                                    <div className=" text-pk-content-secondary">
+                                        {"Triggered " + dayjs().add(-1.2, "days").fromNow()}
+                                    </div>
+                                </div>
+                                <div className="flex gap-1">
+                                    <img
+                                        className="w-5 h-5 rounded-full"
+                                        src={prebuild.commit?.author?.avatarUrl}
+                                        alt="author avatar"
```

Comment:
We can leave it empty, it is only decorative 🙏 

```suggestion
                                        alt=""
```